src/ctapipe/utils/astro.py

```diff
@@ -4,10 +4,151 @@
 not provided by external packages and/or to satisfy particular needs of
 usage within ctapipe.
 """
-from astropy import units as u
+
+import logging
+from pathlib import Path
+
+import astropy.units as u
 from astropy.coordinates import Angle, SkyCoord
+from astropy.table import Table
+from astropy.time import Time
+from astroquery.vizier import Vizier
+
+log = logging.getLogger("main")
+
+__all__ = ["get_bright_stars_with_motion", "get_bright_stars"]
+
+
+CACHE_FILE = Path("~/.psf_stars.ecsv").expanduser()
+
+
+def cut_stars(stars, pointing=None, radius=None, Bmag_cut=None, Vmag_cut=None):
+    """
+    utility function to cut stars based on magnitude and/or location
+
+    Parameters
+    ----------
+    stars: astropy table
+        Table of stars, including magnitude and coordinates
+    pointing: astropy Skycoord
+        pointing direction in the sky (if none is given, full sky is returned)
+    radius: astropy angular units
+        Radius of the sky region around pointing position. Default: full sky
+    Vmag_cut: float
+        Return only stars above a given apparent magnitude. Default: None (all entries)
+    Bmag_cut: float
+        Return only stars above a given absolute magnitude. Default: None (all entries)
+
+    Returns
+    -------
+    Astropy table:
+       List of all stars after cuts with same keys as the input table stars
+    """
+
+    if Bmag_cut is not None and "Bmag" in stars.keys():
+        stars = stars[stars["Bmag"] < Bmag_cut]
+    if Vmag_cut is not None and "Vmag" in stars.keys():
+        stars = stars[stars["Vmag"] < Vmag_cut]
+
+    if radius is not None:
+        if pointing is None:
+            raise ValueError(
+                "Sky pointing, pointing=SkyCoord(), must be "
+                "provided if radius is given."
+            )
+        separations = stars["ra_dec"].separation(pointing)
+        stars["separation"] = separations
+        stars = stars[separations < radius]
+
+    return stars
+
+
+def get_bright_stars_with_motion(
```

Comment:
Do we have a reason to keep the getter without proper motion?

Reply:
No, that's essentially a bug

Reply:
ok, i'll get rid of it